internal/docs/design.md

```diff
@@ -594,13 +597,22 @@ Sample code for the helper for escaping strings:
 // package escape provider helpers for escaping and unescaping strings.
 package escape
 
-// Escape returns s, with all bytes for which shouldEscape returns true
-// escaped.
+// Escape returns s, with all runes for which shouldEscape returns true
+// escaped to "__0xXXX__", where XXX is the hex representation of the rune
```

Comment:
Don't you need four X's?

Reply:
Yes, done. Although it's not always 4 hex digits. But I think the example demonstrates that.